app/src/pages/inside/common/launchFiltersContainer/launchFiltersContainer.jsx

```diff
@@ -90,16 +93,20 @@ export class LaunchFiltersContainer extends Component {
     const currentFilter = this.createFilterQuery(this.getConditions());
 
     if (!isEqual(currentFilter, newFilter)) {
-      this.fetchLaunches(newFilter);
+      this.fetchLaunches({ [PAGE_KEY]: 1, newFilter });
     }
 
+    const conditionsWithFilteringField = addFilteringFieldToConditions(conditions);
+
     if (this.props.activeFilter) {
-      this.updateFilter(this.props.activeFilterId, addFilteringFieldToConditions(conditions));
+      this.updateFilter(this.props.activeFilterId, conditionsWithFilteringField);
     } else {
       this.props.createFilter({
-        conditions: addFilteringFieldToConditions(conditions),
+        conditions: conditionsWithFilteringField,
       });
     }
+
+    this.props.onChange(conditions);
```

Comment:
I see that this handler doesn't have any arguments.
[Code line](https://github.com/reportportal/service-ui/pull/1312/files#diff-cd9d7a96a10c68f93eec371ce8b5b725R354)

Reply:
Right. Argument is not used for now. I just think it's a common pattern to pass new value to `onChange` no matter if someone plans to use it or not.